.github/workflows/ci_cd.yml

```diff
@@ -68,6 +68,32 @@ jobs:
           python-version: ${{ matrix.python-version }}
           # use-python-cache: false
 
+  changelog-check:
+    name: Changelog Update Check
+    if: github.event_name == 'pull_request'
```

Comment:
@RobPasMue  could this be wrapped in a pyansys action?

Reply:
Yes, probably. But we are looking towards a more efficient way of automating the update of CHANGELOG files =) - I talked with @klmcadams and this is more of a "short term" solution. We will implement ``towncrier`` with the Ansys actions and that will give you many more automated features in regards to changelog handling :)